pkg/core/xds/rules.go

```diff
@@ -131,23 +138,25 @@ func BuildRules(list []PolicyItem) Rules {
 			break
 		}
 		// 3. For each combination determine a configuration
-		var conf proto.Message
+		confs := []proto.Message{}
```

Comment:
Should this type be different now if you are distancing yourself from protobufs?

Reply:
yep, I'm replacing `proto.Message` with `ResourceSpec` in the next PR, but I guess I can fix this right here